<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartScreen</class>
 <widget class="QDialog" name="StartScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="LoginForm" name="loginWidget">
      <property name="toolTipDuration">
       <number>1</number>
      </property>
     </widget>
     <widget class="RegistrationForm" name="registerWidget">
      <property name="toolTipDuration">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginForm</class>
   <extends>QWidget</extends>
   <header>sources/loginform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RegistrationForm</class>
   <extends>QWidget</extends>
   <header>sources/registrationform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
